.github/workflows/jest-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Run Jest Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+          cache: 'npm'  # Cache npm dependencies
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run Jest tests
```

Comment:
We can just add this to existing `.github/workflows/nextjs-lint-build.yml` otherwise new workflow would increase build time. Let me know if you see any issues with that?

Reply:
Agreed, will do the needful changes.